programs/mpl-core/src/plugins/update_delegate.rs

```diff
@@ -127,6 +127,22 @@ impl PluginValidation for UpdateDelegate {
         }
     }
 
+    fn validate_update(
+        &self,
+        authority_info: &AccountInfo,
+        authority: &Authority,
+    ) -> Result<ValidationResult, ProgramError> {
+        if authority
```

Comment:
Would this be more robust if we piped the resolved_authorities in and did that comparison instead? In case you want to delegate the Asset Owner as the UA?

Reply:
generally agree with this, it might be useful to pipe in the resolved auths for all validation cycle methods. We can do it in a separate PR

Reply:
Will do in follow on PR